### DISCUSSION TITLE
Allow Libraries Other Than TV Shows/Movies & Anime

### DISCUSSION THREAD

**Discussion by Budlyte** [Archmonger/Conreq#76] [Feature Requests]
If a server utilizes libraries to sort content, such as Kids Movies & TV Shows, there isn't a way to navigate to them for requests.

![image](https://user-images.githubusercontent.com/4451365/131756669-13db92f6-c7a5-4c9f-bb07-ffa156310b4b.png)

**Comment by Archmonger** [comment]
Agreed.

Just a heads up we're in the progress of breaking Conreq into smaller installable apps.

Within V1, I had planned something similar to this within the "Movie/TV Requesting" app.

Let me know what you think.
https://github.com/Archmonger/Conreq/projects/3#card-54613070

**Comment by Budlyte** [comment]
Oh, nice!  I"m guessing separating Requests and Administrator into separate apps.

I like your layout alot more than Overseerr, so I really hope it gets where you want it.
